### ISSUE TITLE
SpellRight: The suggestion was not applied because it is outdated.

### ISSUE THREAD

**Issue by blueray453** [bartosz-antosik/vscode-spellright#157]
When I am trying to fix the spelling with the yellow (light) button, it is giving error 

> SpellRight: The suggestion was not applied because it is outdated.

![vs](https://user-images.githubusercontent.com/855297/39959082-34a9eaaa-562e-11e8-83b3-a03c85ae0566.png)

**Comment by bartosz-antosik** [comment]
Could you please try reproduce this in a smaller document that you can paste here?

**Comment by blueray453** [comment]
When I tried to reproduce in a very small document, it worked.

Let me give you a medium document which does not work.

**Test.adoc**

```
Habit
-----
Make habit by consistency
21 days is scientific. For harder stuff keep doing it until you feel uncomfortable not doing it.
18-254 days to make a habit or give up bad habit.

Focus
-----

Follow one course till u are successfull.... do not diversify
Be consistent

Forgive yourself

Choose what we are


Develop self dicipline

Dicipline is commitment, your first commitment

Self dicipline can transform sorrow to faith

Do not wait till you feel like to do it. The moment might never come. Waiting to feel good to do something is the definition of lazy.

Dicipline is saying yes to 1 decision and no to 99 destructions.

Diciplice is to control primative desires.

Snoozing alarm clock is a sign of indiscipline.

Procastination is lack of dicipline.

You will be how you view yourself

Who I must become to achieve that goal. Become the person in advance.

Act like what you want to be. be what you want to be today.
```

**Comment by bartosz-antosik** [comment]
The following dots are causing this. Probably document size has nothing to do. I will look into this.

**Comment by bartosz-antosik** [comment]
Should be resolved in version 2.4.0.
